features/step_definitions/networking.rb

```diff
@@ -725,14 +727,31 @@
   ensure_admin_tagged
   node = node(node_name)
   host = node.host
-  @result = host.exec_admin("/sbin/bridge vlan show")
+  @result = host.exec_admin("/sbin/bridge -j vlan show")
   raise "Failed to execute bridge vlan show command" unless @result[:success]
+  @result[:parsed] = JSON.load @result[:stdout].to_s
+end
+
+Given /^the number of bridge PVID (\d+) #{QUOTED} VLANs added between the #{SYM} and #{SYM} clipboards is (\d+)$/ do |pvid, mode, clip_a, clip_b, expected_vlans|
+  pvid = pvid.to_i
+  added_bridges = cb[clip_b] - cb[clip_a]
+  logger.info("added_bridges: #{added_bridges}")
+  num_vlans = added_bridges.count {|b|
+    c = b["vlans"].count {|v|
+      v["vlan"] == pvid && v["flags"].include?("PVID") && v["flags"].include?(mode)
+    }
+    c > 0
+  }
```

Comment:
this is the PVID diff check algorithm, needs review.  @anuragthehatter 

we take the added bridges and count all the vlans that match the `PVID` and have flags `"Egress Untagged"`

Reply:
Thanks Ross for the efforts. lgtm except we want to make sure we strictly don't check ```"Egress untagged"``` but just ```"ntagged```. The reason were explained in few of teh old PRs as some platforms exhibits small/capital U and private clusters or external http proxy one's tends to add ```Egress```` so we definitely shouldn't check ```Egress and U/u```

Reply:
This shouldn't be possible in the JSON output based on the source from iproute2

https://git.kernel.org/pub/scm/network/iproute2/iproute2.git/tree/bridge/vlan.c#n421
```c
static void print_vlan_flags(__u16 flags)
{
	if (flags == 0)
		return;

	open_json_array(PRINT_JSON, "flags");
	if (flags & BRIDGE_VLAN_INFO_PVID)
		print_string(PRINT_ANY, NULL, " %s", "PVID");

	if (flags & BRIDGE_VLAN_INFO_UNTAGGED)
		print_string(PRINT_ANY, NULL, " %s", "Egress Untagged");
	close_json_array(PRINT_JSON, NULL);
}
```

Reply:
Well I'll check more, maybe 4.5 is diff

On 4.6 we are 
```
sh-4.4# rpm -qf /usr/sbin/bridge
iproute-5.3.0-1.el8.x86_64
```
Code is the same in v5.3.0

https://git.kernel.org/pub/scm/network/iproute2/iproute2.git/tree/bridge/vlan.c?h=v5.3.0&id=18e631bd4b1269cdc32abd09dc3cd4a3c209e8e2#n447

Reply:
Looks like it was fixed here

2018-09-10 11:48:06 -0700
https://git.kernel.org/pub/scm/network/iproute2/iproute2.git/commit/bridge/vlan.c?h=v5.3.0&id=0f36267485e30099a4f735c3aadfa58b5efa1918

first added in v4.19.0.

Reply:
4.5.0-0.nightly-2020-07-25-031342
```bash
sh-4.4# rpm -qf `which bridge`
iproute-5.3.0-1.el8.x86_64
```

I have a fix using `ntagged`, just need to test

Reply:
Thanks. Nice dissection on iproute, cool